ush/machine/macos.yaml

```diff
@@ -1,18 +1,31 @@
 platform:
-  WORKFLOW_MANAGER: none
+  WORKFLOW_MANAGER: rocoto
   NCORES_PER_NODE: 8
-  SCHED: none
-  RUN_CMD_FCST: 'mpirun -n ${PE_MEMBER01} '
-  RUN_CMD_POST: 'mpirun -n 4 '
+  TASKTHROTTLE: 1
+  SCHED: slurm
+  CCPA_OBS_DIR: /Users/username/DATA/UFS/obs_data/ccpa/proc
+  MRMS_OBS_DIR: /Users/username/DATA/UFS/obs_data/mrms/proc
+  NDAS_OBS_DIR: /Users/username/DATA/UFS/obs_data/ndas/proc
+  DOMAIN_PREGEN_BASEDIR: /Users/username/DATA/UFS/FV3LAM_pregen
+  METPLUS_PATH: ""
+  MET_BIN_EXEC: bin
+  MET_INSTALL_DIR: ""
+  RUN_CMD_FCST: mpirun -n ${PE_MEMBER01}
+  RUN_CMD_POST: mpirun
   RUN_CMD_SERIAL: time
-  RUN_CMD_UTILS: mpirun -n 4
+  RUN_CMD_UTILS: mpirun
   PRE_TASK_CMDS: '{ ulimit -a; }'
```

Comment:
Similar to @mark-a-potts update for ush/machine/linux.yaml, should `ulimit -s unlimited` also be added to line 17 here as well?